lib/services/database_test.go

```diff
@@ -384,46 +385,58 @@ func TestRDSTagsToLabels(t *testing.T) {
 
 // TestDatabaseFromRedshiftCluster tests converting an Redshift cluster to a database resource.
 func TestDatabaseFromRedshiftCluster(t *testing.T) {
-	cluster := &redshift.Cluster{
-		ClusterIdentifier:   aws.String("mycluster"),
-		ClusterNamespaceArn: aws.String("arn:aws:redshift:us-east-1:1234567890:namespace:u-u-i-d"),
-		Endpoint: &redshift.Endpoint{
-			Address: aws.String("localhost"),
-			Port:    aws.Int64(5439),
-		},
-		Tags: []*redshift.Tag{
-			{
-				Key:   aws.String("key"),
-				Value: aws.String("val"),
+	t.Run("success", func(t *testing.T) {
```

Comment:
just moving existing test in a subtest. ignore whitespace may hide all these shifts.